string axisID
string[] paramValue
---
int32 res